Firestore/Swift/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# v7.7.0
```

Comment:
should be 7.8.0

Reply:
I'm fairly confident this was actually shipped in 7.7.0 but we missed the CHANGELOG update. 
![Version 7.7.0 highlighted in the latest Firestore Swift PR](https://user-images.githubusercontent.com/1097316/110031642-2b517800-7d05-11eb-8592-4b4ef55b5410.png)